### DISCUSSION TITLE
How do I push locally cached packages to Nuget using Manifests

### DISCUSSION THREAD

**Discussion by Faaux** [microsoft/vcpkg#19400] [Q&A]
Hey everyone,

I am trying out the binary caching features of VCPKG which work well. To not clutter the remote registry during testing I removed `nuget,REPO,readwrite` from `VCPKG_BINARY_SOURCES`.

Now that everything is working and cached locally I wanted to push/export those packages to Nuget but cant seem to figure out the appropriate command(s).

Current used env var looks like this: `VCPKG_BINARY_SOURCES=nuget,GitLab,readwrite;default`

Any help would be much appreciated.

Best,

Faaux

**Comment by JackBoosY** [comment]
@strega-nil-ms can you please answer this question?

Thanks.

**Comment by Faaux** [comment]
Any update on this? Just ran into this again today :)
